src/main/java/digital/slovensko/autogram/ui/gui/GUIApp.java

```diff
@@ -35,29 +36,42 @@ public void start(Stage windowStage) throws Exception {
             final var params = LaunchParameters.fromParameters(getParameters());
             final var controller = new MainMenuController(autogram, userSettings);
 
-            server = new AutogramServer(autogram, params.getHost(), params.getPort(), params.isProtocolHttps(), cachedExecutorService);
+            if (!params.isStandaloneMode())
+                GUIUtils.startIconified(windowStage);
+
+            windowStage.setTitle("Autogram");
```

Comment:
Na toto treba davat mega bacha. Poradie tychto volani som mega debugoval, sa to spravalo inak na win/linux/macos